sycl/include/CL/sycl/detail/pi.h

```diff
@@ -317,6 +317,7 @@ typedef enum {
   PI_DEVICE_INFO_ATOMIC_MEMORY_SCOPE_CAPABILITIES = 0x11000,
   PI_DEVICE_INFO_GPU_HW_THREADS_PER_EU = 0x10112,
   PI_DEVICE_INFO_BACKEND_VERSION = 0x10113,
+  PI_EXT_ONEAPI_DEVICE_INFO_BFLOAT16 = 0x12000,
```

Comment:
Let's not create more than necessay gaps in numbering. Please use 0x1FFFF

Reply:
Okay thanks. I have updated the value